<ui version="4.0" >
 <class>CAnnulusSegmentParamsGuiComp</class>
 <widget class="QWidget" name="CAnnulusSegmentParamsGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="PositionLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QDoubleSpinBox" name="XSpin" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum" >
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QDoubleSpinBox" name="YSpin" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum" >
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QLabel" name="PositionUnitLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="RadiiLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Radii</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" name="InnerRadiusSpin" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum" >
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QDoubleSpinBox" name="OuterRadiusSpin" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum" >
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="RadiusUnitLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="AnglesLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Angles</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="BeginAngleSB" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum" >
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QDoubleSpinBox" name="EndAngleSB" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum" >
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Â°</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
